src/main/java/DukeException.java

```diff
@@ -0,0 +1,5 @@
+public class DukeException extends Exception {
```

Comment:
Header comments would be good to have for public classes.

Reply:
Thanks! I agree that it should have a header comment since it's a public class so I've added it